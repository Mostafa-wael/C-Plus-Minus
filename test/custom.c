print("Custom");

int a = 90;

print(a);

bool b = !0.0;
print(b);

int x = 0;
int y = 4;

int z = y + a % 2;
print(z);