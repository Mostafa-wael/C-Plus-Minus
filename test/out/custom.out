"Custom"
int
90
bool
1
int
int
